roles/cifmw_cephadm/defaults/main.yml

```diff
@@ -104,3 +104,15 @@ cifmw_cephadm_grafana_container_image: "{{ cifmw_cephadm_container_ns }}/ceph-gr
 cifmw_cephadm_node_exporter_container_image: "{{ cifmw_cephadm_prometheus_container_ns }}/node-exporter:v1.5.0"
 cifmw_cephadm_prometheus_container_image: "{{ cifmw_cephadm_prometheus_container_ns }}/prometheus:v2.43.0"
 cifmw_cephadm_ns: openstack
+cifmw_cephadm_config_key_set_ssl_option: no_sslv2:sslv3:no_tlsv1:no_tlsv1_1
```

Comment:
There is conflict for this file as 1821 is not merged. Please re base the code and resolve the conflicts.

Reply:
I've rebased so that should fix it.